src/docs/guides/fastify.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Deploy a Fastify App
+---
+
+Fastify is a high-performance, low-overhead web framework for Node.js, designed to deliver an exceptional developer experience.
+
+This guide covers how to deploy a Fastify app on Railway in four ways:
+
+1. [One-click deploy from a template](#one-click-deploy-from-a-template).
+2. [From a GitHub repository](#deploy-from-a-github-repo).
+3. [Using the CLI](#deploy-from-the-cli).
+4. [Using a Dockerfile](#use-a-dockerfile).
+
+## One-Click Deploy from a Template
```

Comment:
Maybe also mention ejecting from the template after its deployed?

This creates a clone of the repo the template deployed from in the users github, then any pushes they do to that cloned repo will create a deploy on the service.

Reply:
When you say eject from the template? Do you mean disconnect from the source GitHub repo URL from the service on the dashboard? 🤔

Reply:
@brody192 ☝🏿

Reply:
![image](https://github.com/user-attachments/assets/3b50020b-9892-43f2-9346-91148e878240)
https://docs.railway.app/guides/deploy#eject-from-template-repository

(sorry, gh notifs suck)

Reply:
Thanks. Love learning all these things I didn't know about. 😄